typing/ctype.ml

```diff
@@ -2484,7 +2503,7 @@ let constrain_type_jkind ~fixed env ty jkind =
                  Error
                    (Jkind.Violation.of_ ~jkind_of_type
                       (Not_a_subjkind (ty's_jkind, jkind, sub_failure_reasons)))
-               | Stepped { ty; is_open = is_open2 } ->
+               | Stepped ({ ty; is_open = is_open2 }, (* CR aspsmith: use this? *) _modality) ->
```

Comment:
I wasn't able to find anything that failed due to us throwing this away. Plausibly this is just fine to ignore?

Reply:
This one also seems important. I think it will be triggered with something like

```
type ('a : value mod portable) require_portable
type t = { x : (int -> int) @@ portable } [@@unboxed]
type t2 = t require_portable
```